tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -231,6 +231,33 @@ class CompletionArgSuite extends BaseCompletionSuite {
     ""
   )
 
+  checkSnippet( // see: https://github.com/scalameta/metals/issues/2400
+    "explicit-dollar",
+    """
+      |object Main {
+      |  def test($foo: Int, $bar: Int): Int = ???
+      |  test($f@@)
+      |}
+      |""".stripMargin,
+    """|\$foo = 
```

Comment:
Checking `$` is escaped (otherwise, $foo will be interpreted as a value in vscode's snippet)

Reply:
I think I have already seen escaping `$` by `$$`. Could we stick to double dollars?

Reply:
done https://github.com/scalameta/metals/pull/3686/commits/52903b9480df042f0b99988fc2fae7636848bdd8